### DISCUSSION TITLE
Are there any ways to make `optimize` faster?

### DISCUSSION THREAD

**Discussion by masafumimori** [kernc/backtesting.py#951] [Q&A]
I was wondering if there is a way to make `optimize` faster?

I looked at the code and it already uses `multiprocessing` but when the variety of parameters are a lot (like image) it takes a long time.
So I was wondering if there is a way to make it faster.
<img width="290" alt="スクリーンショット 2023-03-29 125826" src="https://user-images.githubusercontent.com/71799690/228423394-b316805e-d860-4805-b0b5-9d98ebbf25e5.png">

I tried:
- using `multiprocessing`: throws error `daemonic processes are not allowed to have children`
- usnig `ThreadPoolExecutor`: improved a few seconds

If you have any ideas it would be very appreciated.

**Comment by MrDenfish** [comment]
numba may help.
https://numba.readthedocs.io/en/stable/user/5minguide.html
